AIRobot.md

```diff
@@ -0,0 +1 @@
+You've been thinking of adopting one of those new AI robofriends. Yes, that's right. I know you've been lonely of late. [Here's an early present then.](nagma-venture/AIattacks.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.